plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/DefaultGlueColumnStatisticsProvider.java

```diff
@@ -92,17 +98,14 @@ public DefaultGlueColumnStatisticsProvider(AWSGlueAsync glueClient, String catal
             return glueClient.getColumnStatisticsForTable(request);
         }, readExecutor)));
 
-        final ImmutableList.Builder<ColumnStatistics> columnStatsBuilder = ImmutableList.builder();
+        final HiveBasicStatistics tableStatistics = getHiveBasicStatistics(table.getParameters());
+        final ImmutableMap.Builder<String, HiveColumnStatistics> columnStatsMapBuilder = ImmutableMap.builder();
         for (CompletableFuture<GetColumnStatisticsForTableResult> future : getStatsFutures.build()) {
-            final GetColumnStatisticsForTableResult tableColumnsStats = MoreFutures.getFutureValue(future, PrestoException.class);
-            columnStatsBuilder.addAll(tableColumnsStats.getColumnStatisticsList());
+            final GetColumnStatisticsForTableResult tableColumnsStats = MoreFutures.getFutureValue(future, TrinoException.class);
+            tableColumnsStats.getColumnStatisticsList()
+                    .forEach(col -> columnStatsMapBuilder.put(col.getColumnName(), GlueStatConverter.fromGlueColumnStatistics(col.getStatisticsData(), tableStatistics.getRowCount())));
```

Comment:
Add static import for `fromGlueColumnStatistics`.

Also in `getPartitionColumnStatistics`.